app/views/layouts/_header.html.erb

```diff
@@ -24,6 +24,23 @@
           <%= link_to "Terms of Use", terms_path %>
         </a>
       </div>
+      <div>
```

Comment:
Maria can help get this layout so it looks clean. Create a Jira ticket and with the URL and what the expected layout should be in the end.

I also feel that these items should be hidden until the user authenticates.